.github/workflows/ci.yml

```diff
@@ -1,32 +1,28 @@
 name: ForgeRock CI
 on:
-  pull_request_target:
+  pull_request:
 
 env:
   NX_CLOUD_DISTRIBUTED_EXECUTION: true
 
 jobs:
   pr:
     runs-on: macos-latest
-    if: ${{ github.event_name == 'pull_request_target' }}
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{github.event.pull_request.head.repo.full_name}}
           fetch-depth: 0
       - uses: actions/setup-node@v3
+        id: cache
```

Comment:
so the following step can use the cache. node-setup using actions/cache under the hood